internal/handler/httphandler/handler_admin.go

```diff
@@ -30,3 +32,137 @@ func AdminGetUserInfo(resp http.ResponseWriter, req *http.Request) {
 
 	respBody.SetData(info)
 }
+
+func AdminBlockedUser(resp http.ResponseWriter, req *http.Request) {
+	respBody := &HTTPResponse{}
+	defer func() {
+		resp.Write(respBody.Marshall())
+	}()
+
+	var input domain.SetBlockUser
+
+	if err := readBody(req, &input); err != nil {
+		resp.WriteHeader(http.StatusUnprocessableEntity)
+		respBody.SetError(err)
+		return
+	}
+
+	if !input.IsValid() {
+		resp.WriteHeader(http.StatusBadRequest)
+		respBody.SetError(errors.New("invalid input"))
+		return
+	}
+
+	err := service.BlockUser(&input)
+
+	if err != nil {
```

Comment:
в одну строчку можно
```suggestion
	if err := service.BlockUser(&input); err != nil {
```